source/server/filter_chain_manager_impl.cc

```diff
@@ -53,13 +53,30 @@ void FilterChainManagerImpl::addFilterChain(
       const auto& cidr_range = Network::Address::CidrRange::create(destination_ip);
       destination_ips.push_back(cidr_range.asString());
     }
-
+    if (destination_ips.empty()) {
+      if (Network::Address::ipFamilySupported(AF_INET6)) {
+        destination_ips.push_back(Network::Utility::getIpv6CidrCatchAllAddress());
+      }
+      // Insert a 0.0.0.0 to avoid empty entry to match UDS address.
```

Comment:
We use `255.255.255.255` for UDS lookup, so using `0.0.0.0` here might be too wide. Not sure if that's an issue, though.

Reply:
I had the wrong assumption that pipe address. 
However, the code here is correct unless we do need a specialization for UDS some other day